src/store/goods/reducer.ts

```diff
@@ -0,0 +1,46 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import { getGoods } from "../../api";
+import { Good, LOAD_STATUSES_TYPES } from "../../types"
+
+const SLICE_NAME = "goods";
+
+const goodsOnBack = createAsyncThunk(SLICE_NAME, getGoods)
+
+
+interface GoodsStore {
+       loadStatus: string;
+       goods: Good[];
+}
+
+const initialState: GoodsStore = {
+       loadStatus: LOAD_STATUSES_TYPES.SET_UNKNOWN,
+       goods: [{
```

Comment:
для массивов в значении по умолчанию странных данных быть не должно, просто пустой массив.
```suggestion
       goods: []
```